lti_consumer/static/js/xblock_studio_view.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Javascript for LTI Consumer Studio View.
+*/
+function LtiConsumerXBlockInitStudio(runtime, element) {
+    // Run parent function to set up studio view base JS
+    StudioEditableXBlockMixin(runtime, element);
+
+    // Define LTI 1.1 and 1.3 fields
+    const Lti1P1FieldList = [
+        "lti_id",
+        "launch_url"
+    ];
+
+    const Lti1P3FieldList = [
+        "lti_1p3_launch_url",
+        "lti_1p3_oidc_url",
+        "lti_1p3_tool_public_key"
+    ];
+
+    /**
+     * Query a field using the `data-field-name` attribute and hide/show it.
+     *
+     * params:
+     *   field: string. Value of the field's `data-field-name` attribute.
+     *   visible: boolean. `true` shows the container, and `false` hides it.
+     */
+    function toggleFieldVisibility(field, visible) {
+        const componentQuery = '[data-field-name="'+ field + '"]';
+        const fieldContainer = element.find(componentQuery);
+
+        console.log(field);
```

Comment:
remove this before merging

Reply:
Oh, I missed that. Thanks for catching that!

Reply:
Done!